js/services/vCardProperties.js

```diff
@@ -91,16 +91,16 @@ angular.module('contactsApp')
 		impp: {
 			multiple: true,
 			readableName: t('contacts', 'Instant messaging'),
-			template: 'text',
+			template: 'username',
 			defaultValue: {
 				value:[''],
-				meta:{type:['HOME']}
+				meta:{type:['SKYPE']}
 			},
 			options: [
-				{id: 'HOME', name: t('contacts', 'Home')},
-				{id: 'WORK', name: t('contacts', 'Work')},
-				{id: 'OTHER', name: t('contacts', 'Other')}
-			]
+				{id: 'IRC', name:  'IRC'},
+				{id: 'SKYPE', name:'Skype'},
+				{id: 'TELEGRAM', name:'Telegram'}
+    	]
```

Comment:
"103:2  error  Mixed spaces and tabs  no-mixed-spaces-and-tabs" <- this is the travis error